src/client/ui/controls.js

```diff
@@ -138,6 +204,21 @@ export default class StorybookControls extends React.Component {
         <div style={h1WrapStyle}>
           <h3 style={h1Style}>React Storybook</h3>
         </div>
+        <div style={filterTextWrapStyle}>
+          <input
+            style={filterTextStyle}
+            type="text"
+            placeholder="Filter"
+            name="filter-text"
+            value={this.state.filterText}
+            onChange={this.filterStoryList.bind(this)}
```

Comment:
Actually, that's fine for specially this kind of apps since we won't render items in often. (Often means few times a sec)

And js.bind is pretty fast and React does a lot more costly stuff for virtual DOM diffing.